removing tech leads

xmulligan opened the pull request:
it seems we are moving away from having tech leads to this should be reflected in the governance see #131

iawells commented:
I think a better way of describing this is we'll add them when we have something specific for them to do, but I agree that until then we'd be better off not speculating about the process and role.

taylor reviewed (approved):
lgtm

Smitholi67 reviewed (approved):
Lgtm.  Agree that Tech Leads can be added when there becomes a need.

taylor commented:
@xmulligan fixed merge conflict. check to make sure it looks correct (mainly deleted the sections headers you had deleted that were in main)

jeffsaelens reviewed (approved):
Update looks solid.

rannyh reviewed (changes requested):
LGTM

sishbi commented:
LGTM

xmulligan commented on GOVERNANCE.md:
```diff
@@ -55,9 +42,7 @@ See [CONTRIBUTING](CONTRIBUTING.md) documentation for more information how to ge
-### Timeline
-
-Co-chairs and tech leads will be elected for one year terms and may run for reelection.
+Co-chairs will be elected for one year terms and may run for reelection.
```
```suggestion
### Timeline

Co-chairs will be elected for one year terms and may run for reelection.
```

xmulligan commented on GOVERNANCE.md:
```diff
@@ -69,11 +56,9 @@ In other words, if two contributors are employed by Company X, two by Company Y,
-
-### Voting Procedure
+Each organization listed in the [interested parties document](interested-parties.md) has one vote. Interested parties can be added at any time, but must be added at least one week before any election to have a vote. Any contributor from an organization may cast the vote for that organization. Each organization can cast one vote for a co-chair candidate in each of the communities (Kubernetes (K8s) ...
 
```
```suggestion

### Voting Procedure
```

xmulligan commented:
@taylor those weren't supposed to be deleted, but I made this PR before #146 was merged into main so it looks like I deleted them. I've added them back in. Otherwise I think it is ready to merge and it has 5 approvals
